build.gradle

```diff
@@ -26,15 +26,17 @@ repositories {
 }
 
 dependencies {
-    compile 'org.glassfish.jersey.core:jersey-server:2.25'
-    compile 'org.glassfish.jersey.containers:jersey-container-servlet:2.25'
-    compile 'ch.qos.logback:logback-classic:1.1.7'
-    compile 'com.sun.mail:javax.mail:1.5.6'
+    compile 'org.glassfish.jersey.core:jersey-server:2.36'
+    compile 'org.glassfish.jersey.containers:jersey-container-servlet:2.36'
+    compile 'org.glassfish.jersey.inject:jersey-hk2:2.36'
+    compile 'ch.qos.logback:logback-classic:1.2.11'
+    compile 'com.sun.mail:javax.mail:1.6.2'
+    compile 'javax.ws.rs:javax.ws.rs-api:2.1.1'
```

Comment:
For what are all those extra dependencies needed. For example, I see that when doing this we fetch the log4j dependency, while we didn't do that earlier. And we all now the history of log4j of course.

Furthermore, I still get some exceptions while building. An earlier exception disappeared, but it introduces a new one.

```
Exception in thread "main" java.lang.NullPointerException: Cannot invoke method getText() on null object
```

They are not blocking, but it's still a bit weird.